ui/src/app/cohort-search/overview/overview.component.html

```diff
@@ -22,12 +22,12 @@ <h2 id="total-count">
 
   <ng-container *ngIf="chartData$ | async as chartData">
     <div class="card bg-faded">
-      <div class="card-header">
+      <div class="card-header header-text">
         Results by Gender
-        <clr-icon shape="angle"
-          [dir]="showGenderChart ? 'up' : 'down'"
-          (click)="toggleShowGender()">
-        </clr-icon>
+        <!--<clr-icon shape="angle"-->
+          <!--[dir]="showGenderChart ? 'up' : 'down'"-->
+          <!--(click)="toggleShowGender()">-->
+        <!--</clr-icon>-->
```

Comment:
Do we need this commented out section for future use? If not, we should just remove it.